docs/github-action.md

```diff
@@ -0,0 +1,108 @@
+# GitHub Action
+
+To make `ko` even easier to use with [GitHub Actions](https://github.com/features/actions), use [`setup-ko`](https://github.com/marketplace/actions/setup-ko).
+
+This action installs `ko`, logs in using `${{ github.token }}`, and configures `ko` to push to [GHCR](https://github.com/features/packages) by default.
+
+## Example
+
+```yaml
+name: Publish
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+      - uses: ko-build/setup-ko@v0.6
+      - run: ko build
+```
+
+_That's it!_ This workflow will build and publish your code to [GitHub Container Registry](https://ghcr.io).
+
+By default, the action sets `KO_DOCKER_REPO=ghcr.io/[owner]/[repo]` for all subsequent steps, and uses the `${{ github.token }}` to authorize pushes to GHCR.
+
+See [documentation](./configuration.md) to learn more about configuring `ko`.
+
+The action works on Linux and macOS [runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).
+
+## Select `ko` version to install
+
+By default, `setup-ko` installs the [latest release of `ko`](../releases).
+
+You can specify a specific release version:
```

Comment:
I tripped over the wording of "specify a specific"

s/specify/configure/ or s/specific//